### ISSUE TITLE
Reagarding the test set

### ISSUE THREAD

**Issue by abhijithvnair94** [vikramforsk2019/Time-series-Weather-forecasting#1]
Hi Vikram,

If it is possible kindly upload the test dataset also. Or atleasr share the link from where you get both test and training dataset.

Thank you
